test/cluster-tests/kubernetes/k8s-utils.sh

```diff
@@ -66,5 +66,81 @@ function test_linux_deployment() {
 }
 
 function test_windows_deployment() {
-  echo "coming soon"
+  ###### Testing a simpleweb windows deployment
+  log "Testing Windows deployments"
+
+  log "Creating simpleweb service"
+  kubectl apply -f "$DIR/simpleweb-windows.yaml"
+  count=90
+  while (( $count > 0 )); do
+    log "  ... counting down $count"
+    running=$(kubectl get pods --namespace=default | grep win-webserver | grep Running | wc | awk '{print $1}')
+    if (( ${running} == 1 )); then break; fi
+    sleep 10; count=$((count-1))
+  done
+  if (( ${running} != 1 )); then
+    log "K8S: gave up waiting for deployment"
```

Comment:
Two comments:
1. Use unique error strings. To get list of existing strings, run "git grep K8S test/cluster-tests".
2. Update test/acs-engine-test/report/report.go with the new errors.

Thanks!